pkg/tinydtls/Makefile.dep

```diff
@@ -1,9 +1,43 @@
 ifneq (,$(filter tinydtls,$(USEPKG)))
+
   USEMODULE += tinydtls
+
+#Mandatory for tinyDTLS
```

Comment:
Comment should start at the same position (indent with 2 spaces), here and below